docs/BreakingChanges/065 Corner case WPF bug in which databound text box fails is fixed.md

```diff
@@ -0,0 +1,22 @@
+## 65: Corner case WPF bug in which databound text box fails to update is fixed
+
+### Scope
+Transparent
+
+### Version Introduced
+4.5
+
+### Change Description
+TODO
```

Comment:
This breaking change has some 'TODO's for description and recommendation which ought to be filled in.